packages/auto-label/src/auto-label.ts

```diff
@@ -269,10 +279,32 @@ handler.addLabeltoRepoAndIssue = async function addLabeltoRepoAndIssue(
       }
     } else {
       await handler.addLabels(github, owner, repo, issueNumber, [githubLabel]);
+      console.log(`Label added to ${owner}/${repo} for issue ${issueNumber} is ${githubLabel}`);
       wasNotAdded = false;
     }
   }
 
+  let foundSamplesTag = undefined;
+  if (labelsOnIssue) {
+    foundSamplesTag = labelsOnIssue.find(
+      (element: {name: string}) => element.name === 'sample'
```

Comment:
The fact that you need to add the type of `element` makes me think `labelsOnIssue` is an `any` or `any[]`.  Can we put a type on `labelsOnIssue` so you don't need to do the type down here?